qemu/tests/driver_in_use.py

```diff
@@ -85,7 +85,13 @@ def run_subtest(sub_type):
 
         :params: sub_type: Sub test.
         """
-        utils_test.run_virt_sub_test(test, params, env, sub_type)
+        if params.get("bg_stress_test_is_cmd", "no") == "yes":
+            session = vm.wait_for_login()
+            sub_type = utils_misc.set_winutils_letter(
+                session, sub_type)
+            session.sendline(sub_type)
```

Comment:
I think it's better to close session here.

Reply:
Modified.
Thanks